docs/user/geojson.rst

```diff
@@ -0,0 +1,223 @@
+JSON and GeoJSON
+================
+
+GeoMesa provides built-in integration with JSON and GeoJSON. JSON can be stored as an attribute in a simple feature
+and queried using JSONPath through CQL. For users who are not interested in GeoTools, GeoMesa provides a GeoJSON
+API that allows for ingesting and querying with pure JSON.
+
+.. _index_structure:
+
+JSON Attributes
+---------------
+
+GeoMesa allows simple feature attributes to be specified as JSON. JSON attributes must be specified when
+creating a simple feature type:
+
+.. code-block:: java
+
+    import org.locationtech.geomesa.utils.interop.SimpleFeatureTypes;
+
+    // append the json hint after the attribute type, separated by a colon
+    String spec = "json:String:json=true,dtg:Date,*geom:Point:srid=4326"
+    SimpleFeatureType sft = SimpleFeatureTypes.createType("mySft", spec);
+    dataStore.createSchema(sft);
+
+JSON attributes are still strings, and are set as any other strings:
```

Comment:
haha, I'm actually not sure what will happen in that case... I think it will likely fail during serialization. I think I won't mention it for now and just assume a perfect world :)